processor/cumulativetodeltaprocessor/README.md

```diff
@@ -23,8 +23,8 @@ Configuration is specified through a list of metrics. The processor uses metric
 
 The following settings can be optionally configured:
 
-- `include`: List of metrics names or patterns to convert to delta.
-- `exclude`: List of metrics names or patterns to not convert to delta.  **If a metric name matches both include and exclude, exclude takes precedence.**
+- `include`: List of metrics names, patterns or metric types to convert to delta.
+- `exclude`: List of metrics names, patterns or metric types to not convert to delta.  **If a metric name matches both include and exclude, exclude takes precedence.**
```

Comment:
Please include the list of valid metric type values. and mention it is case-insensetive.